pandas/core/arrays/datetimelike.py

```diff
@@ -181,7 +181,7 @@ def _unbox_scalar(self, value: Union[Period, Timestamp, Timedelta, NaTType]) ->
 
         Examples
         --------
-        >>> self._unbox_scalar(Timedelta('10s'))  # DOCTEST: +SKIP
+        >>> _unbox_scalar(Timedelta('10s'))  # doctest: +SKIP
```

Comment:
i think this makes less sense without the `self`, but skipping is OK

Reply:
I agree, let's keep the `self` here (this is an internal docstring and not a runnable example anyway)

Reply:
I was trying to see if this will run without the "self", forgot to revert it.